<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>串口接收到的数据：</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser">
             <property name="minimumSize">
              <size>
               <width>256</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
font: 9pt &quot;宋体&quot;;
color: rgb(0, 255, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>要发送到串口的数据：</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="sendMsgLineEdit">
             <property name="minimumSize">
              <size>
               <width>256</width>
               <height>48</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
font: 9pt &quot;宋体&quot;;
color: rgb(255, 255, 127);</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="sendMsgBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>发送数据</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>参数设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>串 口：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="portNameComboBox">
               <item>
                <property name="text">
                 <string>COM1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM9</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM11</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM12</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>波特率：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="baudRateComboBox">
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>数据位：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="dataBitsComboBox">
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>校验位：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="parityComboBox">
               <item>
                <property name="text">
                 <string>无</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>奇</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>偶</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>停止位：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="stopBitsComboBox">
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>相关操作</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
             <property name="spacing">
              <number>36</number>
             </property>
             <item>
              <widget class="QPushButton" name="clearDataBtn">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>36</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>清空接收.</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="openComBtn">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>36</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>打开串口.</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="closeComBtn">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>36</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>关闭串口.</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
